toolchain/sem_ir/typed_insts.h

```diff
@@ -1059,6 +1059,25 @@ struct PointerType {
   TypeId pointee_id;
 };
 
+// An instruction that indicates a type is required to be complete. This is only
+// created for generic types and produces a witness that the type is complete.
+//
+// TODO: Eventually this should be replaced by a witness for an interface that
+// models type completeness, and should track other information such as the
+// value representation.
+struct RequireCompleteType {
```

Comment:
WDYT of an action phrasing like "FormCompleteType" or "MakeCompleteType", maybe "ConvertDependentToCompleteType"? But, maybe "Require" better reflects how you're viewing what this is doing.

Reply:
I think "require" better reflects how I'm viewing this, when it appears within a generic eval block, at least for now. Once we have template support I think it might become a bit clearer what naming approach we want to use for metaprogramming actions with side effects in the eval block.